.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[env]
+RSTEST_TIMEOUT = "300"
+RUST_TEST_THREADS = "1"
```

Comment:
Just out of curiosity: Am I right to assume that this is for rate limiting?

Reply:
Yes, that's right. The downloader already uses 5 parallel connections by default, so running the tests in parallel on top of that lead to errors (at least on my local machine) due to the many concurrent downloads.